spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/FindHidingSubClassTest.java

```diff
@@ -0,0 +1,178 @@
+package edu.umd.cs.findbugs.detect;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+
+import org.junit.Test;
```

Comment:
The project got migrated from junit vintage to junit jupiter. Can you please modify the import (preferably after a master merge)?

Reply:
Done!